### PR TITLE
Implement cheats

### PR DESCRIPTION
**Pull request by FranciscoDA**
Implements the following cheats:
 * Damage inflicted multiplier
 * Damage received multiplier
 * Stat growth multiplier
 * Infinite ammo
 * Hostiles multiplier

### CODE REVIEW

File: game/ui/city/buildingscreen.cpp [hunk 1]
```diff
@@ -31,17 +31,15 @@ std::shared_future<void> loadBattleBuilding(sp<GameState> state, sp<Building> bu
                                             StateRef<Vehicle> playerVehicle)
 {
 	auto loadTask = fw().threadPoolEnqueue(
-	    [hotseat, building, state, raid, playerAgents, playerVehicle]() -> void {
+	    [hotseat, building, state, raid, playerAgents, playerVehicle]() mutable -> void {
```
Comment (JonnyH) [inline game/ui/city/buildingscreen.cpp]: This should probably be pulled out into another commit

File: game/state/rules/battle/battlemap.cpp [hunk 2]
```diff
@@ -134,10 +135,12 @@ sp<Battle> BattleMap::createBattle(GameState &state, StateRef<Organisation> oppo
 	}
 	auto alienOrg = state.getAliens();
 
+	float hostilesMultiplier = config().getFloat("OpenApoc.Cheat.HostilesMultiplier");
+
 	int countAliens = 0;
 	for (auto &a : *aliens)
 	{
-		for (int i = 0; i < a.second; i++)
+		for (int i = 0; i < (int)(a.second * hostilesMultiplier); i++)
```
Comment (JonnyH) [inline game/state/rules/battle/battlemap.cpp]: Try to avoid c-style casts - preference is for static_cast<int>(whatever) (or an actual rounding function)

File: game/ui/city/buildingscreen.cpp (hunk 1 above)
Comment (FranciscoDA) [reply game/ui/city/buildingscreen.cpp]: I've split this between commits 02bc978 and 439288c

File: game/state/rules/battle/battlemap.cpp (hunk 2 above)
Comment (FranciscoDA) [reply game/state/rules/battle/battlemap.cpp]: I changed these to use std::round, which is more appropriate anyways

**Comment by FilmBoy84** [comment]
I believe all of @JonnyH 's requested changes have been resolved - anything else to do?
I can confirm that menu appears to work as intended in game
Are we good to merge?
